manifests/gcp_marketplace/chart/kubeflow-pipelines/values.yaml

```diff
@@ -17,6 +17,8 @@ images:
 
 gcpSecretName: "user-gcp-sa"
 serviceAccountCredential: ""
+gcsConfigName: "gcs-config"
```

Comment:
Should this name be more specific? E.g. `kfp-default-storage-config`?

Reply:
Done.